Formula/opentsdb.rb

```diff
@@ -0,0 +1,151 @@
+class HbaseLZORequirement < Requirement
+  fatal true
+
+  satisfy(:build_env => false) { !Tab.for_name("hbase").without?("lzo") }
```

Comment:
You can use `Tab.for_name("hbase").with?("lzo")` instead of `!Tab.for_name("hbase").without?("lzo")` to avoid the double negation.